language/stdlib/modules/TransactionFee.move

```diff
@@ -30,40 +31,51 @@ module TransactionFee {
     resource struct LBRIdent<CoinType> { }
 
     /// Called in genesis. Sets up the needed resources to collect
-    /// transaction fees by the `0xB1E55ED` account.
-    public fun initialize(blessed_account: &signer, fee_account: &signer) {
-        Transaction::assert(Signer::address_of(blessed_account) == 0xB1E55ED, 0);
+    /// transaction fees from the `0xFEE` account with the `0xB1E55ED` account.
+    public fun initialize(association: &signer, fee_account: &signer) {
+        Association::assert_is_root(association);
+        Transaction::assert(Signer::address_of(fee_account) == 0xFEE, 0);
+
+        let dummy_auth_key_prefix = x"00000000000000000000000000000000";
```

Comment:
Nit: maybe pass the `dummy_auth_key_prefix` in? That way there's one less "magical" constant.